Jenkinsfiles/Jenkinsfile.cbc-run-multi-pr-checks

```diff
@@ -59,6 +82,24 @@ pipeline {
       }
     }
 
+    stage("START MSLSX in background") {
+      steps{
```

Comment:
Might consider a when condition, if the start MSLSX and BB2 background is only required for selenium tests:
```
when {
        expression { params.RUN_SELENIUM_TESTS == true }
 }
```